rootfs/scheduler/resources/pod.py

```diff
@@ -384,14 +384,20 @@ def _set_image_secret(self, data, namespace, **kwargs):
         data['imagePullSecrets'] = [{'name': secret_name}]
 
     def delete(self, namespace, name):
+        # get timeout info from pod
+        pod = self.pod.get(namespace, name).json()
```

Comment:
I thought about it and at the time I wanted the function it bubbles up to to handle it since `delete()` would cause a `404` if you call that on a non-existent resource